### ISSUE TITLE
PANTHER:PTN000409486 not transmembrane transport

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-annotation#1541]
EXPORTIN-T (a nuclecytoplasmic transporter)
has
	nucleobase-containing compound transmembrane transporter activity

from panther.

Nucleocytoplasmic transport isn't transmembrane, could you let me know the  entries with the original annotations and I'll check?

Thanks

Val

**Comment by pgaudet** [comment]
Hi Val, 

You can find the supporting annotations here: 
http://www.pantree.org/node/annotationNode.jsp?id=PTN000409486

Thanks, Pascale

**Comment by ValWood** [comment]
OK I could only see one annotation (TAIR) dispute raised (see below).
Could this  mapping be deleted (its definitely incorrect and its causing too many errors  for me to weed out any other violators).

SOURCE	ENTITY TYPE	ENTITY ID	GO ID	EVIDENCE	ECO ID	REFERENCE	WITH	EXTENSION
TAIR	protein	Q7PC79	GO:0015932	IGI	ECO:0000316	PMID:12913168	SGD:S000001688	

Dispute ID: 6095
Nature of dispute: This dispute concerns the GO term used
Details:
Hi,

this is an exportin, but it has a molecular function annotation as a transmembrane transporter.

The reason is the comment  on
GO:0006913 Name	nucleocytoplasmic transport:

Note that transport through the nuclear pore complex is not transmembrane because the nuclear membrane is a double membrane, and is not traversed.

**Comment by pgaudet** [comment]
Hi @tberardini , I am tagging you so you can let us know the status of this issue. 

Thanks, Pascale

**Comment by tberardini** [comment]
Internally tracked at TAIR, 3/13/17 based on Protein2GO dispute initiated by Val
[Protein2GO] Notification of annotation dispute concerning protein Q7PC79 (Dispute ID: 6095)

----------------------
Hi Val,

Thanks for the notification.   I've updated the annotation from

GO:0015932, nucleobase-containing compound transmembrane transporter activity

to

GO:0005487, nucleocytoplasmic transporter activity

I tried to do something on protein2go to leave a comment to that effect but can't figure out how.  In any case, the fix will show up with the next file commit on April 1, 2017 (for real :) ).

Thanks,

Tanya

----------

Annotation line from 
gene-associations/gene_association.tair.gz
revision 40125, 20170401

TAIR	locus:2030280	PSD		**GO:0005487**	TAIR:Publication:501710104|PMID:12913168	IGI	SGD:S000001688	F	AT1G72560	AT1G72560|PSD|PAUSED|F28P22.25|F28P22_25	protein	taxon:3702	**20170313**	TAIR		TAIR:locus:2030280
